validation/provisioning/rke2/dynamic_node_driver_test.go

```diff
@@ -0,0 +1,129 @@
+//go:build dynamic
```

Comment:
Leaving the comment here, but not specific to this file. With the edits that you're making to these files, can you make sure that any changes needed to the respective READMEs are made?